javascript/rules/base.js

```diff
@@ -1,30 +1,17 @@
 module.exports = {
-    "extends": [
-        "eslint:recommended"
+    'plugins': [
+        '@stylistic/js',
     ],
 
-    "rules": {
-        "indent": ["error", 4, { "SwitchCase": 1 }],
-        "no-console": ["warn"],
-        "keyword-spacing": ["error", { "before": true, "after": true }],
-        "array-bracket-spacing": ["error", "always"],
-        "object-curly-spacing": ["error", "always"],
-        "comma-dangle": ["warn", ["warn", {
-            "arrays": "always-multiline",
-            "objects": "always-multiline",
-            "imports": "always-multiline",
-            "exports": "always-multiline",
-            "functions": "only-multiline",
-         }]],
-        "no-unused-vars": ["error", { "ignoreRestSiblings": true }],
-        "max-params": ["error"],
-        "brace-style": ["error", "1tbs"],
-    
-        "quotes": ["error", "single"],
-        "no-multi-spaces": ["error", { "ignoreEOLComments": true }],
-        "no-shadow": ["error"],
+    'extends': [
+        'eslint:recommended',
+    ],
 
-        "multiline-ternary": ["error", "always-multiline"],
-        "operator-linebreak": ["warn", "before"],       // This will be later made an error. Done this way to make migration easier.
-    }
```

Comment:
It looks like a bunch of these have been moved into `stylistic.js`. What's the rationale behind the location change?

I'm not sure why some `rules` stay here, but others did not?

Reply:
the rules that stay are ones that are still native to eslint, the ones that moved are the ones that are now part of stylistic. I.e. eslint have deprecated the ones that have moved to use stylistic 
![image](https://github.com/ratehub/code-style/assets/97109156/3fa96138-07ca-40a6-86f0-115680ac3d91)

Reply:
I think its better to seperate the two because they are on different websites and technically a plugin now.